README.rst

```diff
@@ -38,7 +38,7 @@ ODE solvers, and more.
 
 - **Website:** https://docs.scipy.org/doc/scipy/reference/
```

Comment:
Actually we could also modify this one
```suggestion
- **Website:** https://scipy.org
```

As for the doc one (https://docs.scipy.org/), I am not sure what do to.

Reply:
And citing should be https://scipy.org/citing-scipy/